packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -394,7 +394,7 @@ class _AnyTapGestureRecognizer extends BaseTapGestureRecognizer {
   @protected
   @override
   void handleTapUp({PointerDownEvent? down, PointerUpEvent? up}) {
-    onAnyTapUp?.call();
+    invokeCallback('onTapUp', () => onAnyTapUp?.call());
```

Comment:
Looking at the other invocations of invokeCallback, this should probably only be called if onAnyTapUp is non-null?

Reply:
```suggestion
    if (onAnyTapUp != null) {
      invokeCallback('onAnyTapUp', onAnyTapUp!);
    }
```